apis/types/container_config.go

```diff
@@ -303,6 +323,49 @@ func (m *ContainerConfig) validateOnBuild(formats strfmt.Registry) error {
 	return nil
 }
 
+var containerConfigTypeRichModePropEnum []interface{}
```

Comment:
Please see https://github.com/alibaba/pouch/tree/master/apis#generated-api-types

If you need to add functions, I think we had better add them in daemon side, in daemon/mgr or a separated folder in under pouch. @wangforthinker

Reply:
Oh, my fault. No need to update.